### ISSUE TITLE
Clean up eslint configuration

### ISSUE THREAD

**Issue by pete-the-pete** [gabrielcsapo/ponticus#30]
tl;dr: we can have 1 `.estlinrc.json` file for the whole project.

ESlint only runs on JS files by default, and requires `--ext .ts` to run on typescript. If ESlint can't find any files to run, it errors out (unless a [--no-error-on-unmatched-pattern](https://eslint.org/docs/latest/user-guide/command-line-interface#--no-error-on-unmatched-pattern) is used), which causes our [CI to fail](https://github.com/gabrielcsapo/ponticus/actions/runs/3772334516/jobs/6413203283).

This can be fixed by passing `--ext .ts` (or `--ext .ts,.js` for packages that have both JS and TS) in the package.json lint script.

We should create a `.eslintrc.json` at the root level and take advantage of [the cascade](https://eslint.org/docs/latest/user-guide/configuring/configuration-files#cascading-and-hierarchy) to simplify the configuration.
